### PR TITLE
Render meta[description] for service manual topic pages

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

**Comment by thehenster** [comment]
Just a minor comment but looks good. Merge away when you're ready

**Comment by AndrewVos** [comment]
@thehenster yeah it raises http://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Finders#find-instance_method

**Comment by AndrewVos** [comment]
Refactored a bit...

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,6 +8,9 @@
   <!--[if IE 8]><%= stylesheet_link_tag "application-ie8" %><script>var ieVersion = 8;</script><![endif]-->
   <%= csrf_meta_tags %>
   <%= render partial: 'govuk_component/analytics_meta_tags', locals: { content_item: @content_item.content_item } %>
+  <% if content_for(:meta_description).present? %>
```
Comment (benlovell) [inline app/views/layouts/application.html.erb]: `content_for` has a predicate for this.

Comment (thehenster) [inline app/views/layouts/application.html.erb]: TIL http://api.rubyonrails.org/classes/ActionView/Helpers/CaptureHelper.html#method-i-content_for-3F Thanks

**Pull request merged**